eva/interfaces/relational/db.py

```diff
@@ -23,18 +37,43 @@ def __init__(self, reader, writer):
         self._reader = reader
         self._writer = writer
         self._cursor = None
+        self._result: Batch = None
 
     def cursor(self):
         # One unique cursor for one connection
         if self._cursor is None:
             self._cursor = EVACursor(self)
         return self._cursor
 
+    def load(
+        self, file_regex: str, table_name: str, format: str, **kwargs
+    ) -> EVARelation:
+        return self.cursor().load(file_regex, table_name, format, **kwargs)
+
+    def table(self, table_name: str) -> EVARelation:
+        return self.cursor().table(table_name)
+
+    def query(self, sql_query: str) -> EVARelation:
+        return self.cursor().query(sql_query)
+
+    def df(self) -> pandas.DataFrame:
+        if not self._result:
+            raise Exception("No valid result with the current connection")
+        return self._result.frames
+
+    def create_vector_index(
+        self, index_name: str, table_name: str, expr: str, using: str
+    ) -> "EVACursor":
```

Comment:
Why is this a string instead of a type?

Reply:
This is a type from the documentation standpoint. This is a way to refer to a class that hasn't been initialized yet. We can change it if you suggest another approach.